php/class-customize-snapshot-manager.php

```diff
@@ -494,9 +502,6 @@ public function update_snapshot() {
 		if ( ! check_ajax_referer( self::AJAX_ACTION, 'nonce', false ) ) {
 			status_header( 400 );
 			wp_send_json_error( 'bad_nonce' );
-		} elseif ( ! current_user_can( 'customize' ) ) {
```

Comment:
Actually this is what is causing the test failures. Not sure why you want to remove this?